samples/apps/copilot-chat-app/importdocument/README.md

```diff
@@ -32,18 +32,23 @@ Importing documents enables Copilot Chat to have up-to-date knowledge of specifi
 4. **Run** the following command to import a document to the app under the global document collection where
    all users will have access to:
 
-   `dotnet run -- --file .\sample-docs\ms10k.txt`
+   `dotnet run -- --files .\sample-docs\ms10k.txt`
    
    Or **Run** the following command to import a document to the app under a chat isolated document collection where
    only the chat session will have access to:
 
-   `dotnet run -- --file .\sample-docs\ms10k.txt --chat-id [chatId]`
+   `dotnet run -- --files .\sample-docs\ms10k.txt --chat-id [chatId]`
 
    > Note that this will open a browser window for you to sign in to retrieve your user id to make sure you have access to the chat session.
 
    > Currently only supports txt and pdf files. A sample file is provided under ./sample-docs.
 
    Importing may take some time to generate embeddings for each piece/chunk of a document.
+
+   To import multiple files, specify multiple `--files` options. For example:
```

Comment:
this seems odd. I woudld expect only one `--files` option to be required and then a list of files to follow: `dotnet run -- --files ./file1.txt ./file2.txt`.

Reply:
I think the files can be wrapped in double quotes to be intepreted as one argument: `dotnet run -- --files "./file1.txt ./file2.txt."`

Reply:
I actually found a way that will allow just a single `--files`.